kvbc/src/kvbc_app_filter/kvbc_app_filter.cpp

```diff
@@ -308,11 +308,16 @@ uint64_t KvbAppFilter::getValueFromTagTable(const std::string &key) const {
     LOG_ERROR(logger_, msg.str());
     throw std::runtime_error(msg.str());
   }
-  auto result = concordUtils::fromBigEndianBuffer<uint64_t>(val->data.data());
+  std::string delim = kTagTableKeySeparator;
+  std::string result = val->data.data();
+  size_t delim_pos = result.find(delim);
```

Comment:
Effectively, we want to store <uint64>#<uint64>. If we use `toBigEndianStringBuffer` then we know the exact position of each component because this function will always return the same length string for a uint64. Meaning, instead of "finding" which involves more computation, we should be able to index directly into that string. Offset 0 is the global event group id, offset 8 is the separator, and offset 9 is the external event group id.